data_loader/gsc_dataset.py

```diff
@@ -95,6 +95,27 @@ def __init__(self, config):
                 self.labels_by_dataset[dataset] += ([silence_label] * silence_size)
             self.label_mapping[silence_label] = LABEL_SILENCE
 
+        # noise samples
+        self.noise_samples_by_dataset = {
+            DatasetType.TRAIN: [],
+            DatasetType.DEV: [],
+            DatasetType.TEST: []
+        }
+
+        sample_rate = config["sample_rate"]
+        for file_name in noise_files:
+            full_noise = librosa.core.load(file_name, sr=sample_rate)[0]
+            for i in range(0, len(full_noise)-sample_rate, sample_rate):
+                noise_sample = full_noise[i:i + sample_rate] * random.random()
```

Comment:
Why multiply with random.random()?

Reply:
In the [original implementation](https://github.com/castorini/honk/blob/master/utils/model.py#L301),
noise is added as following
```
a = random.random() * 0.1
data = np.clip(a * bg_noise + data, -1, 1)
```
In honk2, I decide to multiply `random.random()` on the sample and multiply 0.1 on retrieval (line 168)